solidity/contracts/converter/interfaces/IConverter.sol

```diff
@@ -44,16 +44,16 @@ interface IConverter is IOwned {
 
     function setConversionFee(uint32 _conversionFee) external;
 
-    function withdrawTokens(
+    function addReserve(IERC20 _token, uint32 _ratio) external;
```

Comment:
Minor: We may as well rename `_ratio` to `_weight`.

Reply:
👍